tests/performance/event_generator/tests/test_logeventgenerator.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (C) 2024, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+"""Unit tests for the LogEventGenerator class."""
+
+from pathlib import Path
+
+import pytest
+
+from event_generator import LogEventGenerator
+
+
+@pytest.fixture
+def setup_log_generator(tmp_path: Path) -> tuple[LogEventGenerator, Path]:
+    """Setup for LogEventGenerator with a temporary directory.
+
+    Args:
+        tmp_path (LocalPath): Provides a temporary directory path fixture from pytest.
+
+    Returns:
+        tuple: A tuple containing an instance of LogEventGenerator and the path to the log file.
+    """
+    path = tmp_path / "test.log"
```

Comment:
Create instead a temporal file as a fixture for each tests to ensure independency

Reply:
Fixed here: https://github.com/wazuh/wazuh-qa/pull/5697/commits/d72c5acab5adf4faab966948a4a1bef368f21c88